### PR TITLE
13.0 mig stock quant manual assign

### PR DESCRIPTION
**Pull request by digitalsatori**

### CODE REVIEW

**Comment by digitalsatori** [comment]
@pedrobaeza Please kindly help. The black's change make flake8 unhappy?

**Comment by pedrobaeza** [comment]
Please strictly follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

**Comment by digitalsatori** [comment]
@pedrobaeza I think I've already followed the migration instruction. I squashed the commits this time. 
Can you please take a look at the travis log? The "black" forced the changes which unsatisfied
flake8 standard(which is not correct IMHO). It seems a deadlock to me.

**Comment by digitalsatori** [comment]
I can't get the log file from the runbot server either. It appears 404 Error when retrieving any log file.

**Comment by pedrobaeza** [comment]
I'm afraid you didn't. There's no commit `[IMP] $MODULE: black, isort, prettier`, and it seems you are not applying it locally. See also https://odoo-community.org/blog/the-oca-blog-1/post/97

**Comment by digitalsatori** [comment]
I hope I didn't miss anything, but I did run `pre-commit run -a`. The thing is 'black' insists to remove the space after the last comma in a list, while the flake8 complain that the space should be there.
So in my second try, I added back the space which removed by 'black' to satisfy flake8, but it failed again in you ci server. It appears to me a deadlock. Please do let me know if I misunderstand anything. Thank you!

git_diff

**Comment by pedrobaeza** [comment]
Well, 2 things:

- There's still missing the commit I'm mentioning that should contain all the auto-formatting burden, and it mustn't be mixed with the migration itself.
- Where is flake8 run? Maybe it's an extra step you have that shouldn't?

**Comment by digitalsatori** [comment]
This what failed on the ci server that is caused by 'black' reformat process
Job__5532_1_-_OCA_stock-logistics-warehouse_-_Travis_CI

**Comment by pedrobaeza** [comment]
@sbidoul are you aware about the fail he is commenting?

@digitalsatori any way, you have to split the commits in 2: one for pre-formatting and another with the migration itself.

**Comment by sbidoul** [comment]
@digitalsatori You need to manually remove the trailing comma when all list/dict items fit on a single line. Black has an open issue to fix that automatically: https://github.com/psf/black/issues/1288

**Comment by digitalsatori** [comment]
thanks @sbidoul for the heads-up. I'm so dumb, I should have removed the trailing comma in the first place. Good experience anyway.

thanks @pedrobaeza , will separate the commit as instructed.

**Comment by digitalsatori** [comment]
wow, all green. Thanks folks @pedrobaeza @sbidoul

**Comment by digitalsatori** [comment]
since all the test went green, can anybody here please review it and merge it if all things are OK. Thank you!

**Comment by digitalsatori** [comment]
@pedrobaeza Would you mind to help with the code review? Thank you!

**Comment by pedrobaeza** [comment]
I'm not able to review it now because I don't have any target customer with this module. You can review other PRs and ask in exchange that they review yours.

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

File: stock_quant_manual_assign/wizard/assign_manual_quants.py [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+        quants_lines = []
+        for quant in available_quants:
+            line = {}
+            line["quant_id"] = quant.id
+            line["on_hand"] = quant.quantity
+            line["location_id"] = quant.location_id.id
+            line["lot_id"] = quant.lot_id.id
+            line["package_id"] = quant.package_id.id
+            line["owner_id"] = quant.owner_id.id
+            line["selected"] = False
```
Comment (ps-tubtim) [inline stock_quant_manual_assign/wizard/assign_manual_quants.py]:
```
line = {
    "quant_id": quant.id,
    "on_hand": quant.quantity,
    "location_id": quant.location_id.id,
    "lot_id": quant.lot_id.id,
    "package_id": quant.package_id.id,
    "owner_id": quant.owner_id.id,
    "selected": False,
}
```
Why not you assign `line` like this?
IMHO, I think it better.

**Review by ps-tubtim — COMMENTED** [review]
I have some questions about it.

File: stock_quant_manual_assign/wizard/assign_manual_quants.py (hunk 1 above)
Comment (digitalsatori) [reply stock_quant_manual_assign/wizard/assign_manual_quants.py]: thanks @ps-tubtim, good catch! the code is updated as suggested.

**Review by ps-tubtim — APPROVED** [review]
Code LGTM.

**Pull request merged**